index.js

```diff
@@ -1,65 +1,99 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
+ console.log("%cQuestion 1", "color: red") 
+console.log("blob")
 
-// console.log(counter) 
+var counter = 1
+var counter = 2
+ 
 // // => 2
-// console.log("%c----------", "color: red") 
+ console.log("%c----------", "color: red") 
 
 
 // ***** Question 2 *****
 // *** Uncomment the lines below to test
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
+ 
+const name = "Shafee"
 
-// console.log(name)
-// => "Raffy"
-// name = "Not Raffy" 
-// => TypeError
-// console.log("%c----------", "color: red") 
+console.log("%cQuestion 2", "color: red") 
+
+console.log(name)
+
+ // => "Raffy"
+ // name = "Not Raffy" 
+ // => TypeError
+ console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 3 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
+ 
+function drinkWater(currentThirstLevel) {
+
+  console.log("Man I sure am thirsty")
+  currentThirstLevel -= 1
+  console.log("Ahh that hits the spot")
+  return currentThirstLevel
+}
+
+
+  
 
-// console.log(drinkWater(12))
+console.log("%cQuestion 3", "color: red") 
+
+ console.log(drinkWater(12))
 // // => "Man I sure am thirsty"
 // // => "Ahh that hits the spot"
 // // => 11
-// console.log("%c----------", "color: red") 
+ console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 4 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 4", "color: red") 
+ 
 
-// console.log(sameSameButDifferent(5, 5)) 
+console.log("%cQuestion 4", "color: red") 
+
+function sameSameButDifferent(num,maybeNum){
+if (num === maybeNum){return "same same"}
+if (num == maybeNum){return "same same (but different)"}
+if (num != maybeNum){return "different"}
+} 
+console.log(sameSameButDifferent(5, 5)) 
 // // => "same same"
 
-// console.log(sameSameButDifferent(123, "123")) 
+ console.log(sameSameButDifferent(123, "123")) 
 // // => "same same (but different)"
 
-// console.log(sameSameButDifferent(5, 7)) 
+ console.log(sameSameButDifferent(5, 7)) 
 // // => "different"
 
-// console.log(sameSameButDifferent(123, "122")) 
+ console.log(sameSameButDifferent(123, "122")) 
 // // => "different"
-// console.log("%c----------", "color: red") 
+
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 5 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 5", "color: red")
-// const student1 = { name: "Duane", grade: 88 }
-// updateGrade(student1, 92)
-// console.log(student1)
+ console.log("%cQuestion 5", "color: red")
+ 
+ function updateGrade(student,new_grade){
+   student.grade = new_grade 
+ return student
+  }
```

Comment:
Good. Just remember to use camelCase in JavaScript and again, watch that indentation
```suggestion
 function updateGrade(student,newGrade){
   student.grade = newGrade 
   return student
  }
```